torch/distributions/uniform.py

```diff
@@ -30,6 +31,10 @@ class Uniform(Distribution):
     def mean(self):
         return (self.high + self.low) / 2
 
+    @property
+    def mode(self):
+        return nan * torch.empty_like(self.high)
```

Comment:
Why not just return `self.mean`. That's a valid maximal value, right?  My guess is that more users would prefer us to chose an arbitrary maximal value than would prefer us to get nit picky about whether they're asking for maximal or maximum.